### PR TITLE
Document new field from the es database

### PR DESCRIPTION
**Pull request by miiu96**
#### Description of the pull request (what is new / what has changed)
- Added information about the new fields from the Elasticsearch database.

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/indices/accounts.md [hunk 1]
```diff
@@ -17,14 +17,17 @@ The `_id` field of this index is represented by a bech32 encoded address.
...
+| shardID          | The shardID field represents the shard where the address belongs to, based on its bytes.                                                                                             |
+| timestamp        | The timestamp field represents the last moment when the address balance was changed.                                                                                                 |
+| developerRewards | The developerRewards represents the fees that were accumulated after all the smart contract calls that can be claimed by the owner.                                                  |
+| currentOwner     | The currentOwner field holds the address in a bech32 format of the current owner of the smart contract. This field is populated only for the smart contract addresses.               |
+| userName         | The userName field contains the heretag the address possesses.                                                                                                                       |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:30]: herotag

File: docs/sdk-and-tools/indices/accounts.md [hunk 2]
```diff
@@ -17,14 +17,17 @@ The `_id` field of this index is represented by a bech32 encoded address.
...
+| balanceNum       | The balanceNum field holds the amount of EGLD the address possesses, in a numeric format. Example: 1.5.                                                                              |
+| nonce            | The nonce field represents the sequence number of the address.                                                                                                                       |
+| shardID          | The shardID field represents the shard where the address belongs to, based on its bytes.                                                                                             |
+| timestamp        | The timestamp field represents the last moment when the address balance was changed.                                                                                                 |
+| developerRewards | The developerRewards represents the fees that were accumulated after all the smart contract calls that can be claimed by the owner.                                                  |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/accounts.md:28]: The developerRewards represents the fees that were accumulated after all the smart contract calls. They can be claimed by the owner.

File: docs/sdk-and-tools/indices/delegators.md [hunk 3]
```diff
@@ -23,6 +23,18 @@ The `_id` field of this index is composed in this way: `blake2bHash(delegatorAdd
 | contract       | This field holds the bech32 encoded address of the staking provider contract to whom it was delegated to.                            |
 | activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated nor unbondable).                                    |
 | activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated nor unbondable), in a numeric format. Example: 1.5. |
+| unDelegateInfo | The unDelegateInfo contains a list with data about the unDelegated values.                                                           |
+| timestamp      | The timestamp field represents the last moment when was a interation with the delegation contract.                                   |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/delegators.md:27]: The timestamp field represents the last moment of an interaction with the delegation contract.

File: docs/sdk-and-tools/indices/accounts.md (hunk 1 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md:30]: done

File: docs/sdk-and-tools/indices/accounts.md (hunk 2 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/accounts.md]: done

File: docs/sdk-and-tools/indices/delegators.md (hunk 3 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md]: done

**Review by bogdan-rosianu — APPROVED** [review]

File: docs/sdk-and-tools/indices/accountsesdt.md [hunk 4]
```diff
@@ -17,17 +17,18 @@ The `_id` field of this index is composed in this way: `{bech32address}_{tokenId
...
+| balanceNum | The balanceNum field holds the amount of ESDT tokens the address possesses, in a numeric format.                                      |
+| data       | The data field is a structure that contains extra data about a token, such as the creator of an NFT.                                  |
+| tokenNonce | The tokenNonce field holds the sequence number of the token. This field is empty in the case of `FungibleESDT`.                       |
+| token      | The token field holds the name of the token.                                                                                          |
+| timestamp  | The timestamp field represents the timestamp when the address balance was changed.                                                    |
+| type       | The type field represents the type of the ESDT token. It can be `FungibleESDT`, `NonFungibleESDT`, `SemiFungibleESDT`, or `MetaESDT`. |
+| frozen     | The frozen field is true if the current ESDT token the address possesses is frozen.                                                   |
```
Comment (schimih) [inline docs/sdk-and-tools/indices/accountsesdt.md:31]: typo

**Review by schimih — APPROVED** [review]

**Pull request merged**